jenkins-config/clouds/openstack/Primary/ubuntu18.04-docker-4c-16g.cfg

```diff
@@ -0,0 +1,2 @@
+Image: ZZCI - Ubuntu 18.04 - docker - x86_64 - 20200925-233259.113
```

Comment:
This should be IMAGE_NAME

Reply:
Thanks, I also realized it should use `=` too, not `:`. Just re-pushed.

Reply:
You seem to have re-pushed without including the fix...

Reply:
@#$&*! sorry about that, re-pushed, and verified this time.